### PR TITLE
Exceptions in jsonpath templates

### PR DESCRIPTION
**Pull request by ratulm**

### CODE REVIEW

File: projects/batfish-client/src/main/java/org/batfish/client/BfCoordWorkHelper.java [hunk 1]
```diff
@@ -575,6 +575,35 @@ public String getObject(String containerName, String testrigName, String objectN
...
+
+      addTextMultiPart(multiPart, CoordConsts.SVC_KEY_API_KEY, _settings.getApiKey());
+
+      JSONObject jObj = postData(webTarget, multiPart);
+      if (jObj == null) {
+        return null;
+      }
+
+      if (!jObj.has(CoordConsts.SVC_KEY_QUESTION_LIST)) {
+        _logger.errorf("question list key not found in: %s\n", jObj.toString());
```
Comment (dhalperi) [inline projects/batfish-client/src/main/java/org/batfish/client/BfCoordWorkHelper.java]: drop `.toString()` -- `%s` should already handle that conversion.

(And better yet, also handles `null` values; tho in this case that's not an issue)

File: projects/coordinator/src/main/java/org/batfish/coordinator/PoolMgrService.java [hunk 2]
```diff
@@ -36,6 +37,33 @@ public JSONArray getInfo() {
   }
 
   @GET
+  @Path(CoordConsts.SVC_RSC_POOL_GET_QUESTION_TEMPLATES)
```
Comment (dhalperi) [inline projects/coordinator/src/main/java/org/batfish/coordinator/PoolMgrService.java]: why do the work manager and pool manager have overlapping APIs?

(code looks identical or at least very close to work mgr?)

File: projects/batfish-common-protocol/src/main/java/org/batfish/common/util/CommonUtil.java [hunk 3]
```diff
@@ -339,6 +339,14 @@ public static void deleteIfExists(Path path) {
     return differenceSet;
   }
 
+  public static boolean equalsWithNullCheck(Object o1, Object o2) {
```
Comment (dhalperi) [inline projects/batfish-common-protocol/src/main/java/org/batfish/common/util/CommonUtil.java]: drop -- this is just the built-in Java Objects.equals(o1, o2)

File: tests/basic/jsonpath-assertion-equals.ref [hunk 4]
```diff
@@ -0,0 +1,81 @@
+{
```
Comment (dhalperi) [inline tests/basic/jsonpath-assertion-equals.ref]: somehow I doubt this stack trace was desired?

File: projects/coordinator/src/main/java/org/batfish/coordinator/PoolMgrService.java (hunk 2 above)
Comment (ratulm) [reply projects/coordinator/src/main/java/org/batfish/coordinator/PoolMgrService.java]: It is identical. I'll explain the goals/rational in person tomorrow. There might be better ways of it. 

FYI, in this comment and the previous one, changes from PR #425 are bleeding in.

**Comment by ratulm** [comment]
approved in person; so forcing close.

**Pull request merged**